test/server/controllers/book.order.aggregation.spec.js

```diff
@@ -2,6 +2,7 @@ var chai = require('chai');
 var expect = chai.expect;
 var should = chai.should();
 
+var Promise = require('bluebird');
```

Comment:
aah interesting.. Promise is built in to node.js (latest versions and io.js) and I think I'm using that on my machine.. thats why I didn't get any error..